.circleci/config.yml

```diff
@@ -30,64 +33,90 @@ commands:
           key: android=tools-v1-{{ checksum "scripts/install-android-tools.sh" }}-{{ arch }}
           paths:
             - ~/android-tools
-
-  create-launch-android-emulator:
-    description: "create and launch android emulators"
-    steps:
-      - run:
-          name: create AVD
-          command: |
-            echo "no" | avdmanager --verbose create avd --force \
-              --name "Pixel_3a_API_26" \
-              --package "system-images;android-26;google_apis_playstore;x86"
-
-      - run:
-          name: start AVD
-          command: emulator @Pixel_3a_API_26 -no-window -no-audio
-          background: true
-
-      - run:
-          name: wait for emulator
-          command: |
-            adb wait-for-device shell 'while [[ -z $(getprop dev.bootcomplete) ]]; do sleep 1; done;'
   
   npm-dependencies:
-    description: "installs all global and local npm dependencies"
+    description: "Install all global and local npm dependencies"
     steps:
       - run: npm install -g cordova
       - run: npm install -g github:apache/cordova-paramedic
+      - run: npm install --legacy-peer-deps
+
+  npm-ios-dependencies:
+    description: "Install all ios global and local npm dependencies"
+    steps:
       - run: npm install -g ios-deploy
-      - restore_cache:
-          keys:
-            - v2-dependencies-{{ checksum "package.json" }}
-            - v2-dependencies-
-      - run: npm install
-      - save_cache:
-          paths:
-            - node_modules
-          key: v2-dependencies-{{ checksum "package.json" }}
```

Comment:
Path `node_modules` didn't exist with new Android machine image but these are install globally so I don't think this cache would have caught them anyway 🤔